knowledge-base/stackhawk/hawkscan-action/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'StackHawk HawkScan Action'
+github-token:
+  action-input:
+    input: githubToken
+    is-default: true
+  permissions:
+    security-events: write
+    security-events-reason: to upload Code Scanning Alert info
```

Comment:
Please keep reason to all lower case letters in this case, e.g. `to upload code scanning alerts`